### ISSUE TITLE
Ensure all init scripts are updated for AppleTalk support.

### ISSUE THREAD

**Issue by NJRoadfan** [Netatalk/netatalk#1547]
Netatalk 2.x tended to use monolithic init scripts to start up all the various services. One thing that guaranteed was that atalkd (if installed) was loaded before afpd, which allowed file sharing over AppleTalk to function. So far, only the systemd scripts have been updated to support this. The OpenRC scripts can likely support this by adding `use atalkd` in the `depend()` section of the `openrc.netatalk` script. The NetBSD scripts appear the same as Netatalk 2.x and likely need no changes. Do FreeBSD, OpenBSD, and Solaris still have working AppleTalk support?

**Comment by rdmark** [comment]
FreeBSD and OpenBSD sadly both did away with their appletalk kernel modules about a decade ago. I made a note of this in the AppleTalk README: https://github.com/Netatalk/netatalk/blob/main/doc/README.AppleTalk

On Solaris we used to depend on the kernel module that was distributed with the netatalk codebase. We removed it from 2.x a few months ago in https://github.com/Netatalk/netatalk/issues/781 because it was hopelessly bitrotted (and SPARC architecture only.)

I'm still holding out hope that we will see a userland appletalk stack in our lifetime. :)
